ggml/src/ggml-sycl/common.hpp

```diff
@@ -32,153 +31,135 @@
 #include "ggml-common.h"
 #pragma clang diagnostic pop
 
-void* ggml_sycl_host_malloc(size_t size);
-void ggml_sycl_host_free(void* ptr);
+void * ggml_sycl_host_malloc(size_t size);
```

Comment:
There are quite a few changes in this file that don't seem to be related to oneDNN cleanup code. I'm not sure whether this is due to a conflict or some clang-format changes. Can you try and reduce the diff?

Reply:
I was working on an old branch initially and I think there have been some formatting changes since, so when I rebased this happened, fixed it in latest update.